common/src/main/java/io/thorntail/openshift/ts/common/arquillian/OpenShiftUtil.java

```diff
@@ -22,10 +22,14 @@
         this.oc = oc;
     }
 
-    private List<Pod> listPodsForDeploymentConfig(String deploymentConfigName) {
+    public List<Pod> listPodsForDeploymentConfig(String deploymentConfigName) {
+        return listPodsForLabel("deploymentconfig",deploymentConfigName);
+    }
+
+    public List<Pod> listPodsForLabel(String labelName, String labelValue) {
```

Comment:
Why this change?

Reply:
This change in OpenShiftUtil was (and probably will be) useful..but after refactoring - this is no longer used, reverted.
modified to 2.5.0.Final
spaces aligned by IDEA reverted.